### ISSUE TITLE
Create cfd trading offer screens

### ISSUE THREAD

**Issue by holzeis** [itchysats/10101#87]
![Image](https://user-images.githubusercontent.com/382048/199581663-564858bf-66e0-4de8-9d53-7b777d8317b0.png)

**Comment by da-kami** [comment]
Initial placeholder screen was added in #96 
Note: Navigation is not depicted correctly at the moment, but introduction of `go_router` should fix that. This ticket is not blocked by this, but navigation may be a bit weird at the moment, because the `CFD Trading` screen is treated as child of the Wallet `Dashboard`.

**Comment by holzeis** [comment]
created a PR for the first screen https://github.com/itchysats/10101/pull/112

**Comment by holzeis** [comment]
I split the second part (cfd order overview) of that ticket into a separate issue - https://github.com/itchysats/10101/issues/115
